db/migrate/20170807233222_create_events.rb

```diff
@@ -0,0 +1,4 @@
+class CreateEvents < ActiveRecord::Migration
+  def change
+  end
```

Comment:
Please complete these migrations so you and/or I can test out these associations!